pycbc/workflow/jobsetup.py

```diff
@@ -251,6 +251,9 @@ def sngl_ifo_job_setup(workflow, ifo, out_files, curr_exe_job, science_segs,
             for pnum, parent in enumerate(sorted_parents):
                 if len(curr_parent) != 1:
                     tag = ["JOB%d" %(pnum,)]
+                    bank_tag = [t for t in parent.tags if 'bank' in t]
+                    # add splitbank tags into the current executable
+                    curr_exe_job.update_current_tags(bank_tag)
```

Comment:
If curr_exe_job has any tags *other* than a bank_tag, I think this would remove them. Need a little bit more to pass existing tags through.

This would also have a JOBX tag (used only in the output file) and the BANKX tag. So we'd have files called INSPIRAL_BANK1_JOB1. (but BANK and JOB are always the same number). Don't think we need *both* here.

Reply:
BANK and JOB are not always the same - JOB numbers are set according to a sort on the tag string a few lines above, meaning that e.g. BANK11 would be JOB4, which will come before BANK2 (which would be JOB13)

Reply:
I think my objection still holds though if this is a one-to-one mapping. If BANK2 is *always* JOB13, then we don't need the JOB13 tag at all(?)